workflows/core/python-service/dispatch_release.yaml

```diff
@@ -117,7 +117,7 @@ jobs:
           tags: cloudforet/${{ github.event.repository.name }}:${{ env.VERSION }}
           build-args: |
             PACKAGE_VERSION=${{ env.PACKAGE_VERSION }}
-            BRANCH_NAME=${{ env.BRANCH_NAME }} # Only for console-api-v2`
+            BRANCH_NAME=${{ env.BRANCH_NAME }}
```

Comment:
Oh...This comment is causing an error...?

Reply:
yeap..  :sweat_smile: